python/python/bystro/ancestry/tests/test_train.py

```diff
@@ -1,10 +1,65 @@
 """Tests for ancestry model training code."""
+import pandas as pd
+from pandas.testing import assert_frame_equal
 
-from bystro.ancestry import train
+from bystro.ancestry.train import VARIANT_REGEX, _parse_vcf_from_file_stream
 
-## this is not a very useful test, really just testing that train
-## loaded successfully, didn't fail on import
 
+def test_VARIANT_REGEX():
+    assert VARIANT_REGEX.match("chr1:123456:A:A")
```

Comment:
Can we include match tests where ref and alt are different instead of the same? We probably want to throw an error when ref=alt in the variant ID maybe as an assert in the train code for the processed vcf if there isn't one yet.

Reply:
good catch!